frontend/pages/admin/IntegrationsPage/cards/MdmSettings/AndroidMdmPage/components/TurnOffAndroidMdmModal/TurnOffAndroidMdmModal.tsx

```diff
@@ -32,7 +35,7 @@ const TurnOffAndroidMdmModal = ({
       onExit();
```

Comment:
We generally don't close modals when the submission fails I think?

Reply:
Thanks! @ghernandez345, do you recall if there was any special reason we'd want to close this particular modal on failure instead of leaving it open?